pkg/monitoring/alloy/templates/alloy-config.alloy.template

```diff
@@ -3,15 +3,20 @@ logging {
   format = "logfmt"
 }
 
+remote.kubernetes.secret "credentials" {
+  namespace = "kube-system"
```

Comment:
This operator is creating this secret so we should pass the namespace here as a variable.

Reply:
@TheoBrigitte the operator is creating a secret on the MC, this is mounted in the WC so the 2 namespaces are not the same

Reply:
Right, so is this the namespace of the alloy app ? do we have this information here ?

Reply:
Well we know it's kube-system and we don't have a reason to change it

Reply:
```suggestion
  # secret is created in the namespace the chart is déployed to via https://github.com/giantswarm/alloy-app/blob/a3238d832a42a93055928f2e164ac0e6cea0dd13/helm/alloy/templates/secret.yaml#L10. in our case it is always kube-system
  namespace = "kube-system"
```

Reply:
@TheoBrigitte i'll fix the build but what do you think about this?

Reply:
Yeah good enough